less/index.less

```diff
@@ -53,7 +53,145 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-}
+	html, body {
+		height: 100%;
+		font-size: 62.5%;
+	
+	}
+
+
+	
+
+	//Variable
+	@main-background:url(../steel.jpg);
+	@text-color:black(20, 19, 19, 0.945);
+	@font:'Audiowide', cursive;
+	//Mixins
+	.round-borders{
+		width:7rem;
+		border-radius: 100%;
+		border:1px solid @text-color;
+		margin-right: 1rem;
+	}
+	.border{
+		border:2px solid black;
+	}
+
+	//start of my css
+
+	.container{
+		background-image: @main-background;
+		background-repeat: no-repeat;
+		background-size: cover;
+		margin:auto;
+		max-width: 880px;
+		border: 2px rgba(0, 0, 0, 0.24) solid;
+		font-size: 2rem;
+		font-family: @font;
+		margin: 10px;
+		color: @text-color;
+		h1{
+			font-size: 4rem;
+			padding-bottom: 1rem;
+
+		}
+		h2{
+			font-size:3rem; 
+			padding-bottom: 1rem;
+			.img1{
+				.round-borders;
+			}
+		}
+		
+
+	}
+	nav{
+		display: flex;
+		justify-content: flex-end;
+		a{
+			text-decoration:none;
+			color:white ;
+			font-size: 1.5rem;
+			padding: 10px 2rem;
+			a:hover{
+				text-decoration: underline;
+				color:rgb(41, 37, 37);	
+			}
```

Comment:
I think you'd want to use `&:hover` instead here. The following:
```
a {
  a:hover {
  
  }
}
```

will look for an `<a>` tag inside an `<a>` tag.

The following are the same, however:

```
a {

}

a:hover {

}
```

and:

```
/* This is for taking advantage of Less features */
a {
  &:hover{

  }
}
```